_rules/SC1-2-1-audio-transcript.md

```diff
@@ -27,15 +24,15 @@ A play button is an interactive element that when activated, plays the audio. Th
 
```

Comment:
I think this is a separate rule, not part of the applicability.

Reply:
Discussed this f2f. Resolved by rewording the applicability.